backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -173,6 +186,8 @@ private UUID addShopToRepository(String shopOwnerId, String shopName, String sho
     private void addTokenListToShop(UUID shopId) {
         List<Token> tokens = new ArrayList<>();
         tokens.add(new Token(UUID.randomUUID(), shopId, 1, Status.ACTIVE));
+        tokens.add(new Token(UUID.randomUUID(), shopId, 1, Status.EXPIRED));
```

Comment:
No need to change now, but would recommend you to read up on Builder pattern.

Reply:
okay will definitely look into that!